mobile/src/reducers/organization/employee.model.ts

```diff
@@ -99,16 +99,26 @@ export class Employee {
         return hasher.value;
     }
 
+    //----------------------------------------------------------------------------
     public getSurname(): string {
         const names = this.name.split(' ');
 
         return names.length > 0 ? names[0] : this.name;
     }
 
+    //----------------------------------------------------------------------------
     public getName(): string {
         const names = this.name.split(' ');
 
         return names.length > 1 ? names[1] : this.name;
     }
 
+    //----------------------------------------------------------------------------
+    public getRoomTitle(): string {
```

Comment:
I'm a bit concerned that there are 2 places with room number string generation. Maybe it's better to extract this somewhere?

Reply:
Sounds reasonable, but where?

Reply:
That's a good question. Some kind of RoomFormatter?

Reply:
Maybe pass fully constructed room title into CurrentPeopleRoomImpl as a nav param would be a solution? I see CurrentPeopleRoomImpl is the only place where room title is constructed besides this function

Reply:
You should still provide some default value if there is no required parameter in payload in this case

Reply:
@DesmanLead Take a look at latest changes, please